ui/src/app/pages/homepage/homepage.tsx

```diff
@@ -390,24 +402,47 @@ export const Homepage = withUserProfile()(class extends React.Component<
                               <RecentWorkspaces />
                             </FlexColumn>
                             <FlexColumn>
-                              <SmallHeader>Recently Accessed Items</SmallHeader>
-                              <RecentResources/>
+                              {this.state.userHasWorkspaces ? (<React.Fragment>
+                                <SmallHeader>Recently Accessed Items</SmallHeader>
+                                <RecentResources/>
+                              </React.Fragment>) : <div style={{backgroundColor: colors.lightBackground,
+                                color: colors.primary,
+                                borderRadius: 10,
+                                padding: '10px 30px',
+                                margin: '30px 0px'}}>
+                                <h2 style={{fontWeight: 600, marginTop: 0}}>Here are some tips to get you started:</h2>
+                                <CustomBulletList>
+                                  <CustomBulletListItem bullet='→'>
+                                    Create a <a href='https://support.google.com/chrome/answer/2364824'>Chrome Profile </a>
```

Comment:
The text looks a bit small compared to the mock, but maybe that's just the screenshot. The mocks have the text at 14px, which I think is the smallest we go.
This also applies to the text:
```
As you create your workspaces, this area will store your most recent workspaces. 
To see all workspaces created, click on See all Workspaces to the right.
```